lib/msal-core/src/authority/Authority.ts

```diff
@@ -33,6 +33,25 @@ export class Authority {
         this.tenantDiscoveryResponse = authorityMetadata;
     }
 
+    public static isAdfs(authorityUrl: string): boolean {
+        const components = UrlUtils.GetUrlComponents(authorityUrl);
+        const pathSegments = components.PathSegments;
+
+        if (pathSegments.length && pathSegments[0].toLowerCase() === Constants.ADFS) {
```

Comment:
Can we just return the check's result?

```js
        return (pathSegments.length && pathSegments[0].toLowerCase() === Constants.ADFS);
```

Saves us the if and the return true/false.

Reply:
Done